docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -34,6 +34,38 @@ image::images/cases-ui-connector.png[Shows the page for creating connectors]
 - {kibana-ref}/swimlane-action-type.html[{swimlane} connector]
 - {kibana-ref}/cases-webhook-action-type.html[{webhook-cm} connector]
 
+[float]
+[[mapped-case-fields]]
+=== Mapped case fields
+
+To represent an {es-sec} case in an external system, {es-sec} case fields are mapped to existing fields in {sn}, {jira}, {ibm-r}, and {swimlane}. Once fields are mapped, you can push updates to external systems and mapped fields are overwritten or appended. Retrieving data from external systems is not supported.
+
+|===
+
+| *Case field* | *Mapped field*
+
+| Title
+
+a| The case `Title` field is mapped to corresponding fields in external systems. Mapped field values are overwritten when you push updates.
+
+* *{sn}*: Mapped to the `Short description` field.
+* *{jira}*: Mapped to the `Summary` field.
+* *{ibm-r}*: Mapped to the `Name` field.
+* *{swimlane}*: Mapped to the `Description` field.
+
+| Description
+| The case `Description` field is mapped to the `Description` field in {sn}, {jira}, {ibm-r}, and {swimlane}. Mapped field values are overwritten when you push updates.
+
+| Comments
+
+a| For {sn} connctors, the case `Comments` field is mapped to the `Work Notes` field in {sn}.
+
+For {jira}, {ibm-r}, and {swimlane} connectors, the case `Comments` field is mapped to the `Comments` field in {jira}, {ibm-r}, and {swimlane}.
```

Comment:
This does involve a bit of repetition with the `Comments` field, but it's repeating meaningful information, not filler phrases.
```suggestion
a| The case `Comments` field is mapped to corresponding fields in external systems.

* *{sn}*: `Work Notes`
* *{jira}*: `Comments`
* *{ibm-r}*: `Comments`
* *{swimlane}*: `Comments`
```

Reply:
I considered this but thought it'd be too overbearing with all the other text, which you removed here :)

Reply:
@stephmilovic similar question here. Do we need to mention that the `Comments` field for the Webhook - Case Management connector likely maps to a `Comments` field in the external system the user is connecting to? Or, is that too much of an assumption to make? If it is, maybe we can omit that information all together.

Reply:
Not making changes here. Adding more info about the webhook cm connector to the section intro.